packages/extensionmanager/src/widget.tsx

```diff
@@ -577,25 +732,30 @@ export class ExtensionView extends VDomRenderer<ListModel> {
         );
       } else {
         searchContent.push(
-          <ListView
-            key="search-items"
-            // Filter out installed extensions:
-            entries={model.searchResult.filter(
-              entry => model.installed.indexOf(entry) === -1
-            )}
-            numPages={pages}
-            onPage={value => {
-              this.onPage(value);
-            }}
-            performAction={this.onAction.bind(this)}
-          />
+          <>
+            <ListView
+              key="search-items"
+              listMode={model.listMode}
+              viewType={'searchResult'}
+              // Filter out installed extensions:
+              entries={model.searchResult.filter(
+                entry => model.installed.indexOf(entry) === -1
+              )}
+              numPages={pages}
+              onPage={value => {
+                this.onPage(value);
+              }}
+              performAction={this.onAction.bind(this)}
+            />
+          </>
```

Comment:
Need to keep at top level so react doesn't complain about keys not being found. I tried this locally.

```suggestion
          <ListView
            key="search-items"
            listMode={model.listMode}
            viewType={'searchResult'}
            // Filter out installed extensions:
            entries={model.searchResult.filter(
              entry => model.installed.indexOf(entry) === -1
            )}
            numPages={pages}
            onPage={value => {
              this.onPage(value);
            }}
            performAction={this.onAction.bind(this)}
          />
```

Reply:
Thx Saul for the fix. Committed your suggestion.